cea/optimization/master/summarize_network.py

```diff
@@ -202,7 +202,7 @@ def calc_temp_withlosses(t0, Q, m, cp, case):
         else:
             t1 = t0 - Q / (m * cp)
     else:
-        t1 = 0
+        t1 = 273
```

Comment:
can we use a constant here? (also, wasn't it more like 273.15?) `ZERO_DEGREES_CELSIUS_IN_KELVIN`?